core/__tests__/models/property/localCache.ts

```diff
@@ -10,10 +10,18 @@ describe("models/property", () => {
 
   beforeAll(async () => {
     await helper.factories.properties();
+    source = await Source.findOne({ where: { state: "ready" } });
+
+    const deletedProperty = await helper.factories.property(
+      source,
+      { key: "deletedProp" },
+      { column: "col" }
+    );
+    await deletedProperty.update({ state: "deleted" });
+
     originalPropertyCount = await Property.scope(null).count({
-      where: { state: "ready" },
+      where: { state: ["ready", "deleted"] },
```

Comment:
Umm... I didn't know there was a shorthand for `Op.In`!

Reply:
[Yeah](https://sequelize.org/master/manual/model-querying-basics.html#shorthand-syntax-for--code-op-in--code-)!